# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../muttprint/no-css.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- muttprint-0.72d/doc/manual/de/Makefile.vanilla	2006-10-04 14:05:50.000000000 +0000
+++ muttprint-0.72d/doc/manual/de/Makefile	2006-10-04 14:05:55.000000000 +0000
@@ -91,7 +91,6 @@
 	mkdir -p $(docdir)/$(docdirname)/manual-$(lang)
 	# ignore the stylesheet-images/ directory because the files are not needed
 	$(INSTALL) manual-$(lang)/*.html $(docdir)/$(docdirname)/manual-$(lang)/
-	$(INSTALL) manual-$(lang)/*.css $(docdir)/$(docdirname)/manual-$(lang)/
 
 gzip:
 	$(gzip) *.ps *.txt
--- muttprint-0.72d/doc/manual/en/Makefile.vanilla	2006-10-04 14:06:19.000000000 +0000
+++ muttprint-0.72d/doc/manual/en/Makefile	2006-10-04 14:06:23.000000000 +0000
@@ -91,7 +91,6 @@
 	mkdir -p $(docdir)/$(docdirname)/manual-$(lang)
 	# ignore the stylesheet-images/ directory because the files are not needed
 	$(INSTALL) manual-$(lang)/*.html $(docdir)/$(docdirname)/manual-$(lang)/
-	$(INSTALL) manual-$(lang)/*.css $(docdir)/$(docdirname)/manual-$(lang)/
 
 gzip:
 	$(gzip) *.ps *.txt
--- muttprint-0.72d/doc/manual/es/Makefile.vanilla	2006-10-04 14:06:10.000000000 +0000
+++ muttprint-0.72d/doc/manual/es/Makefile	2006-10-04 14:06:12.000000000 +0000
@@ -91,7 +91,6 @@
 	mkdir -p $(docdir)/$(docdirname)/manual-$(lang)
 	# ignore the stylesheet-images/ directory because the files are not needed
 	$(INSTALL) manual-$(lang)/*.html $(docdir)/$(docdirname)/manual-$(lang)/
-	$(INSTALL) manual-$(lang)/*.css $(docdir)/$(docdirname)/manual-$(lang)/
 
 gzip:
 	$(gzip) *.ps *.txt
--- muttprint-0.72d/doc/manual/it/Makefile.vanilla	2006-10-04 14:06:31.000000000 +0000
+++ muttprint-0.72d/doc/manual/it/Makefile	2006-10-04 14:06:33.000000000 +0000
@@ -91,7 +91,6 @@
 	mkdir -p $(docdir)/$(docdirname)/manual-$(lang)
 	# ignore the stylesheet-images/ directory because the files are not needed
 	$(INSTALL) manual-$(lang)/*.html $(docdir)/$(docdirname)/manual-$(lang)/
-	$(INSTALL) manual-$(lang)/*.css $(docdir)/$(docdirname)/manual-$(lang)/
 
 gzip:
 	$(gzip) *.ps *.txt
--- muttprint-0.72d/doc/manual/sl/Makefile.vanilla	2006-10-04 14:06:41.000000000 +0000
+++ muttprint-0.72d/doc/manual/sl/Makefile	2006-10-04 14:06:43.000000000 +0000
@@ -91,7 +91,6 @@
 	mkdir -p $(docdir)/$(docdirname)/manual-$(lang)
 	# ignore the stylesheet-images/ directory because the files are not needed
 	$(INSTALL) manual-$(lang)/*.html $(docdir)/$(docdirname)/manual-$(lang)/
-	$(INSTALL) manual-$(lang)/*.css $(docdir)/$(docdirname)/manual-$(lang)/
 
 gzip:
 	$(gzip) *.ps *.txt
